flocker/provision/_ssh/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+from ._model import (
+    Run, run, run_from_args,
+    Sudo, sudo, sudo_from_args,
+    Put, put,
+    Comment, comment,
+    RunRemotely,
```

Comment:
I avoided importing from `_conch` or `_crochet` since the package builders import this. That should probably be fixed in the dockerfiles for the builds.